### DISCUSSION TITLE
Intuition of 0.5 scale mask

### DISCUSSION THREAD

**Discussion by ShunchiZhang** [Jingkang50/OpenPSG#28] [Q&A] [answered]
Hi, would anyone like to explain the purpose or intuition of 0.5 scale mask here?

The explanation of this setting doesn't seem to appear in the PSG paper.

Any responses are highly appreciated :)

https://github.com/Jingkang50/OpenPSG/blob/255545243b15cc12f132fb4b102b8b4662fd681d/configs/psgtr/psgtr_r50_psg.py#L213

https://github.com/Jingkang50/OpenPSG/blob/255545243b15cc12f132fb4b102b8b4662fd681d/openpsg/models/frameworks/psgtr.py#L126-L128

**Comment by GSeanCDAT** [comment, marked as answer]
Hi Shunchi, thanks for your interest in our work! 
The purpose of using half scale mask here is simply to speed up the training process and reduce GPU memory usage. 
You can also use the full-scale mask instead and there might be some improvements in model's performance;)
